akka-server-app/build.sbt

```diff
@@ -0,0 +1,31 @@
+enablePlugins(JavaServerAppPackaging)
+
+name := "mukis-akka-server-app"
+
+version := "1.0"
+
+val akkaVersion = "2.3.9"
+
+libraryDependencies ++= Seq(
+  "com.typesafe.akka" %% "akka-kernel" % akkaVersion,
+  "com.typesafe.akka" %% "akka-actor" % akkaVersion
+)
+
+mainClass in Compile := Some("de.mukis.AkkaServerApp")
+
+maintainer in Linux := "Nepomuk Seiler <nepomuk.seiler@mukis.de>"
+
+packageSummary in Linux := "Custom application configuration"
```

Comment:
This setup is primarily for daemons on Linux or services on Windows. For this reason, I have chosen the name ...Server... The rpm:packageBin works für this setup perfect. The rpm can be installed and the daemon is started automatically The other example should illustrate a command line program.